src/main/java/com/github/jcustenborder/kafka/connect/snmp/SnmpTrapSourceTask.java

```diff
@@ -87,6 +100,11 @@ public void start(Map<String, String> settings) {
 
     this.snmp = new Snmp(this.messageDispatcher, this.transport);
     this.snmp.addCommandResponder(this);
+
+    if (this.config.mpv3Enabled) {
+      setupMpv3(this.snmp, this.config, this.securityProtocols);
```

Comment:
Here we probably need more checks if we default to MPV3?

Reply:
As discussed this will be kept as it is at the moment.